{"name": "test_login_3[2]", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <scripts.test_fixture.TestLogin object at 0x000001A83A819E10>, data = 2\n\n    def test_login_3(self, data):\n        \"\"\"记住密码登录\"\"\"\n        allure.attach(\"输入用户名\", \"输入用户名username_3\")\n        print(\"输入用户名\")\n        allure.attach(\"输入密码\", \"输入用户名password_3\")\n        print(\"输入密码\")\n        allure.attach(\"点击登录\", \"登录成功\")\n        print(\"点击登录\")\n        print(data)\n>       assert False\nE       assert False\n\nscripts\\test_fixture.py:40: AssertionError"}, "description": "记住密码登录", "attachments": [{"name": "输入用户名username_3", "source": "4cd5efe4-aa91-44a6-a551-49fce978fa47-attachment.attach"}, {"name": "输入用户名password_3", "source": "58ffffce-0641-4a17-a4bd-d182cffe6c9a-attachment.attach"}, {"name": "登录成功", "source": "3ef300ec-b16c-4156-9416-3a5fa82452ea-attachment.attach"}], "parameters": [{"name": "data", "value": "2"}], "start": 1575025869177, "stop": 1575025869188, "uuid": "24abed7f-55eb-4c6b-8723-62002cf4c5af", "historyId": "3fa21cf732c1a3ca798649c5798ea1f0", "testCaseId": "133a8e3ccd283ba22e290157fe855949", "fullName": "scripts.test_fixture.TestLogin#test_login_3", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_fixture"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-47A72RD"}, {"name": "thread", "value": "12696-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_fixture"}]}